projects/ng-swiper/src/lib/swiper/swiper.component.ts

```
@@ -0,0 +1,52 @@
+  imports: [],
+  templateUrl: './swiper.component.html',
+  styleUrl: './swiper.component.scss',
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  host: {
+    '[id]': 'id',
+    class: 'ng-swiper swiper',
+  }
+})
+export class SwiperComponent {
+  static nextId: number = 0;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)